src/index.js

```diff
@@ -51,13 +57,14 @@ const mergeImages = (sources = [], options = {}) => new Promise(resolve => {
 
 			if (options.Canvas && options.format === 'image/jpeg') {
 				// Resolve data URI for node-canvas jpeg async
-				return new Promise(resolve => {
+				return new Promise((resolve, reject) => {
 					canvas.toDataURL(options.format, {
 						quality: options.quality,
 						progressive: false
 					}, (err, jpeg) => {
 						if (err) {
-							throw err;
+							reject(err);
```

Comment:
How come this is changed to reject instead of throw?

Reply:
the error was being thrown from an asynchronous function, meaning it would not result in the promise being rejected.

The function provided to `Promise` will reject automatically if it catches an exception, but it won't catch it from something asynchronous.